wca-regulations.md

```diff
@@ -141,8 +141,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3j2) Definition: Two pieces are similar to each other if they are identical in shape and size, or mirrored in shape and identical in size.
     - 3j3) Corrugated/textured parts which permit the orientation of pieces to be distinguished by feel are not permitted for blindfolded events.
 - 3k) Puzzles should be approved by the WCA Delegate before use in the competition.
-    - 3k1) If a puzzle is found to be not permitted during a round, the competitor must submit a replacement.
-    - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF). Exception: If a puzzle is found to be not permitted during a round, earlier results may be replaced with an extra attempt, at the discretion of the WCA Delegate.
+    - 3k1) If it is found during a round that a competitor used a puzzle that is not permitted, the competitor must submit a replacement.
+    - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF). The following exceptions apply:
+      - 3k2a) If a non-permitted puzzle is found before a round is complete, affected results in the round may be replaced with extra attempts, at the discretion of the WCA Delegate.
```

Comment:
Sorry for not being clear with indent += 2.
We have a standard that every level of nesting adds 4 spaces in front of the regulation number, while here 3k2 and 3k2a differ only by 2 spaces. Because of that Travis is showing an error.